requirements.txt

```diff
@@ -10,6 +10,8 @@ djmail==0.11.0
 futures==3.0.3
 gunicorn==19.7.1
 lxml==4.1.0
+mock==2.0.0
```

Comment:
why did you add in these packages? you don't appear to have used `mock` or `pbr` anywhere in your tasks